docs/book/src/forc/plugins/forc_client/forc_call.md

```diff
@@ -0,0 +1,307 @@
+# Forc Call
+
+`forc-call` is a command-line tool for interacting with deployed Fuel contracts. It allows you to make contract calls, query contract state, and interact with any deployed contract on the Fuel network - all from your command line!
+
+The `forc call` command is part of the Forc toolchain and is installed alongside other Forc tools.
+
+## Getting Started
+
+Here are a few examples of what you can do with `forc call`:
+
+Call a simple addition function on a deployed contract (in dry-run mode):
+
+```bash
+forc call 0xe18de7c7c8c61a1c706dccb3533caa00ba5c11b5230da4428582abf1b6831b4d \
+  --abi ./out/debug/counter-contract-abi.json \
+  add 1 2
+```
+
+Query the owner of a deployed DEX contract on testnet:
+
+```bash
+forc call \
+  --testnet \
+  --abi https://raw.githubusercontent.com/mira-amm/mira-v1-periphery/refs/heads/main/fixtures/mira-amm/mira_amm_contract-abi.json \
+  0xd5a716d967a9137222219657d7877bd8c79c64e1edb5de9f2901c98ebe74da80 \
+  owner
+```
+
+## Usage
+
+The basic syntax for `forc call` is:
+
+```bash
+forc call [OPTIONS] --abi <ABI-PATH/URL> <CONTRACT_ID> <SELECTOR> [ARGS]...
+```
+
+Where the following arguments are required:
+
+- `CONTRACT_ID` is the ID of the deployed contract you want to interact with
+- `ABI-PATH/URL` is the path or URL to the contract's JSON ABI file
+- `SELECTOR` is the function name (selector) you want to call
+- `ARGS` are the arguments to pass to the function
+
+## CLI Reference
+
+<details>
+  <summary><b>Forc Call CLI reference</b></summary>
+
+```sh
+forc call --help
+```
+
+```output
+Perform Fuel RPC calls from the comfort of your command line
+
+Usage: forc call [OPTIONS] --abi <ABI> <CONTRACT_ID> <FUNCTION> [ARGS]...
+
+Arguments:
+  <CONTRACT_ID>
+    The contract ID to call
+
+  <FUNCTION>
+    The function signature to call. When ABI is provided, this should be a selector (e.g. "transfer") When no ABI is provided, this should be the full function signature (e.g. "transfer(address,u64)")
+
+  [ARGS]...
+    Arguments to pass into main function with forc run
+
+Options:
+  --abi <ABI>
+    Path or URI to a JSON ABI file
+
+  --node-url <NODE_URL>
+    The URL of the Fuel node to which we're submitting the transaction. If unspecified, checks the manifest's `network` table, then falls back to `http://127.0.0.1:4000`
+    
+    You can also use `--target`, `--testnet`, or `--mainnet` to specify the Fuel node.
+    
+    [env: FUEL_NODE_URL=]
+
+  --target <TARGET>
+    Use preset configurations for deploying to a specific target.
```

Comment:
what is forc call deploying?

Reply:
nothing; it was using existing struct - which had original use in forc-deploy i believe.
Have updated the docs there and in the book:
https://github.com/FuelLabs/sway/pull/6791/commits/66451e764ad93b27fb6d712fbc625369e5ea74b8